src/commands.rs

```diff
@@ -667,13 +657,13 @@ pub fn config_get(ctx: &mut Context<'_>) -> anyhow::Result<()> {
     println!(
       ctx,
       r#"Config:
-  numlock binding:          {nl}
-  capslock binding:         {cl}
-  scrollock binding:        {sl}
+  num lock binding:         {nl}
+  caps lock binding:        {cl}
+  scroll lock binding:      {sl}
```

Comment:
Would it make sense to move these changes into the "Rename numlock, capslock, scrollock options" commit?